public/app/plugins/panel/candlestick/fields.test.ts

```diff
@@ -1,12 +1,12 @@
 import { createTheme, toDataFrame } from '@grafana/data';
 
 import { prepareCandlestickFields } from './fields';
-import { CandlestickOptions, VizDisplayMode } from './models.gen';
+import { PanelOptions, VizDisplayMode } from './panelcfg.gen';
```

Comment:
Can you re-export all the generated types from the `types.ts` file instead. This should be the preferred approach since it gives us one source of truth for all the type imports.

Reply:
Sure thing, will do that!